examples/dnsdoctor/dnsdoctor.lua

```diff
@@ -0,0 +1,77 @@
+--
+-- SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+-- DNS Doctoring : Rewrite DNS type A record to a private address for local clients
+
+local xt = require("xtable")
+local linux = require("linux")
+local string = require("string")
+local action = xt.action
+local family = xt.family
+
+local udp = 0x11
+local dns = 0x35
+
+local function nop() end
+
+local function get_domain(skb, off)
+	local name = ""
```

Comment:
why do you need this? you are only using it to prepend `""` in the return value.. it doesn't make much sense.. perhaps I'm missing something..